testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1517,17 +1519,30 @@
 
   raise 'Error getting server certificate serial!' unless result_code.zero?
 
-  command_minion = "openssl x509 --noout --text -in /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT | grep -A1 'Serial' | grep -v 'Serial'"
   targets = %w[proxy sle_minion ssh_minion rhlike_minion deblike_minion build_host kvm_server]
   targets.each do |target|
     # get all defined minions from the environment variables and check their certificate serial
     next unless ENV.key? ENV_VAR_BY_HOST[target]
+    # Red Hat-like and Debian-like minions store their certificates in a different location
+    certificate = if target == 'rhlike_minion'
+                    '/etc/pki/ca-trust/source/anchors/RHN-ORG-TRUSTED-SSL-CERT'
+                  elsif target == 'deblike_minion'
+                    '/usr/local/share/ca-certificates/susemanager/RHN-ORG-TRUSTED-SSL-CERT.crt'
+                  else
+                    '/etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT'
+                  end
```

Comment:
I think this step can be improved, we should not rely on the name of a target, but instead have another way to identify the kind of OS.
The reasoning behind it, is that this step will not work on Build Validation testsuite, if we ever want to use it there, we will need to fully refactor how we process these targets.

Reply:
Thank you for the suggestion, Oscar. I implemented this with only the CI in mind. However, I already pushed a solution, so the step can also be used in BV.